### PR TITLE
Fixed arvix import

### LINKED ISSUE #150: Troubleshooting OGBN-Arxiv Model: Error with Marius Preprocessing Command in Docker
**Describe the bug**
Hello,

I encountered an issue after building the image and attempting to use the OGBN-Arxiv model. When running the following command within the Marius Docker container from the root directory ("/working_dir/marius"):

`marius_preprocess --dataset ogbn_arxiv --output_directory datasets/ogbn_arxiv_example/
`

It shows:
```
Using existing dataset of OGBN_ARXIV
Traceback (most recent call last):
  File "/usr/local/bin/marius_preprocess", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.10/dist-packages/marius/tools/marius_preprocess.py", line 167, in main
    dataset.preprocess(
  File "/usr/local/lib/python3.10/dist-packages/marius/tools/preprocess/datasets/ogbn_arxiv.py", line 80, in preprocess
    converter = TorchEdgeListConverter(
TypeError: TorchEdgeListConverter.__init__() got an unexpected keyword argument 'columns'
```
Could you please assist me in resolving this issue?

Thank you! 

**Environment**
cmake:
  version: 3.27.7
cpu_info:
  num_cpus: 112
  total_memory: 110GB
cuda:
  version: '11.8'
gpu_info:
  - memory: 44GB
    name: NVIDIA A40
marius:
  bindings_installed: true
  install_path: /usr/local/lib/python3.10/dist-packages/marius
  version: 0.0.2
openmp:
  version: '201511'
operating_system:
  platform: Linux-4.18.0-513.5.1.el8_9.x86_64-x86_64-with-glibc2.35
pybind:
  PYBIND11_BUILD_ABI: _cxxabi1011
  PYBIND11_COMPILER_TYPE: _gcc
  PYBIND11_STDLIB: _libstdcpp
python:
  deps:
    numpy_version: 1.26.2
    omegaconf_version: 2.3.0
    pandas_version: 2.1.3
    pip_version: 22.0.2
    pyspark_version: N/A
    pytest_version: N/A
    torch_version: !!python/object/new:torch.torch_version.TorchVersion
      - 2.0.1+cu118
    tox_version: N/A
  version: 3.10.12 (main, Jun 11 2023, 05:26:28) [GCC 11.4.0]
pytorch:
  install_path: /usr/local/lib/python3.10/dist-packages/torch
  version: !!python/object/new:torch.torch_version.TorchVersion
    - 2.0.1+cu118

### PR DESCRIPTION
**Pull request by sarda-devesh**
Resolves #150

### CODE REVIEW

File: examples/python/fb15k_237_gpu.py [hunk 1]
```diff
@@ -1,9 +1,8 @@
 from pathlib import Path
 
-from omegaconf import OmegaConf
-
 import marius as m
 from marius.tools.preprocess.datasets.fb15k_237 import FB15K237
+from omegaconf import OmegaConf
```
Comment (shivaram) [inline examples/python/fb15k_237_gpu.py:5]: are these changes necessary for the bug fix (moving the import order) or are they auto generated somehow? If it is the latter can you remove them from this PR?

Comment (sarda-devesh) [reply examples/python/fb15k_237_gpu.py]: I ran `tox -e autoformat` for linting and it made those changes as I only updated the `ogbn_arxiv.py` file

Comment (sarda-devesh) [reply examples/python/fb15k_237_gpu.py]: I had created an conda environment and installed tox using pip install inside of that environment. I am assuming a different version of tox got installed which lead to this different behaviour

**Review by JasonMoho — APPROVED** [review]
